docs/wallets/ledger-wallet.mdx

```diff
@@ -19,4 +19,215 @@ for:
 - Transferring tokens
 - Signing transactions
 
+:::important
+
 At this time, Hotspot management actions such as onboarding or asserting location are not supported.
+
+:::
+
+## How To Access Ledger Account in Helium Mobile Wallet
+
+So let's take a look at how to access your Ledger account via the Helium Mobile Wallet app.
+
+### Step 1: Preparation
+Before you begin, there are prerequsites and software you need to install which includes:
+
+* Helium Ledger Nano X Device
+* [Ledger Live Application](https://www.ledger.com/ledger-live)
+* Helium Mobile Wallet app
+
+Nano X users need to have Bluetooth connectivity to manage HNT using the app.
+
+### Step 2: Install the Solana app on your Ledger device
+
+:::important
+
+Helium migration to Solana
+
+Helium blockchain migrated to Solana blockchain. The migration to Solana took place on April 18, 2023, beginning at 9am PST. After the migration the Helium blockchain is officially halted. Learn more
+
+As a result, the Helium app was removed from Ledger Live. After the migration, users should be able to manage their HNT tokens with the Solana app. 
```

Comment:
Should include a link to the ledger migration guide for people who had ledgers before April 18th, 2023

Reply:
Thanks for pointing out, already added the link!